service/src/main/kotlin/no/nav/su/se/bakover/service/ServiceBuilder.kt

```diff
@@ -4,26 +4,78 @@ import no.nav.su.se.bakover.client.Clients
     private val clients: Clients
 ) {
-    fun build() = Services(
-        avstemmingService = AvstemmingServiceImpl(
-            repo = databaseRepos.avstemmingRepo,
-            publisher = clients.avstemmingPublisher
-        ),
-        utbetalingService = StartUtbetalingerService(
-            repo = databaseRepos.objectRepo,
-            simuleringClient = clients.simuleringClient,
-            utbetalingPublisher = clients.utbetalingPublisher
+    fun build(): Services {
+        val utbetalingService = UtbetalingServiceImpl(
+            repo = databaseRepos.utbetalingRepo,
         )
-    )
+        val søknadService = SøknadServiceImpl(
+            søknadRepo = databaseRepos.søknadRepo
+        )
+        val sakService = SakServiceImpl(
+            sakRepo = databaseRepos.sakRepo
+        )
+        return Services(
+            avstemmingService = AvstemmingServiceImpl(
+                repo = databaseRepos.avstemmingRepo,
+                publisher = clients.avstemmingPublisher
+            ),
+            utbetalingService = utbetalingService,
+            oppdragService = OppdragServiceImpl(
+                repo = databaseRepos.oppdragRepo
+            ),
+            behandlingService = BehandlingServiceImpl(
+                behandlingRepo = databaseRepos.behandlingRepo,
+                hendelsesloggRepo = databaseRepos.hendelsesloggRepo,
+                beregningRepo = databaseRepos.beregningRepo,
+                oppdragRepo = databaseRepos.oppdragRepo,
+                simuleringClient = clients.simuleringClient,
+                utbetalingService = utbetalingService,
+                oppgaveClient = clients.oppgaveClient,
+                utbetalingPublisher = clients.utbetalingPublisher,
+                søknadService = søknadService,
+                sakService = sakService
+            ),
+            sakService = sakService,
+            søknadService = søknadService,
+            stansUtbetalingService = StansUtbetalingService(
+                simuleringClient = clients.simuleringClient,
+                utbetalingPublisher = clients.utbetalingPublisher,
+                utbetalingService = utbetalingService
+            ),
+            startUtbetalingerService = StartUtbetalingerService(
+                simuleringClient = clients.simuleringClient,
+                utbetalingPublisher = clients.utbetalingPublisher,
+                utbetalingService = utbetalingService,
+                sakService = sakService
+            )
+        )
+    }
 }
 
 data class Services(
     val avstemmingService: AvstemmingService,
-    val utbetalingService: StartUtbetalingerService
+    val utbetalingService: UtbetalingService,
+    val oppdragService: OppdragService,
+    val behandlingService: BehandlingService,
+    val sakService: SakService,
+    val søknadService: SøknadService,
+    val stansUtbetalingService: StansUtbetalingService,
+    val startUtbetalingerService: StartUtbetalingerService
```

Comment:
Siden klassen allerede heter `Services`, kanskje vi kunne ha forenklet til
```suggestion
    val avstemming: AvstemmingService,
    val utbetaling: UtbetalingService,
    val oppdrag: OppdragService,
    val behandling: BehandlingService,
    val sak: SakService,
    val søknad: SøknadService,
    val stansUtbetaling: StansUtbetalingService,
    val startUtbetalinger: StartUtbetalingerService
```